code/modules/mob/living/carbon/xenomorph/castes/drone/abilities_drone.dm

```diff
@@ -3,20 +3,107 @@
 	plasma_cost = 75
 	acid_type = /obj/effect/xenomorph/acid/weak
 
-/datum/action/xeno_action/activable/salvage_plasma
-	name = "Salvage Plasma"
+/datum/action/xeno_action/activable/salvage_biomass
+	name = "Salvage Biomass"
 	action_icon_state = "salvage_plasma"
-	ability_name = "salvage plasma"
-	var/plasma_salvage_amount = PLASMA_SALVAGE_AMOUNT
-	var/salvage_delay = 3 SECONDS
-	var/max_range = 1
+	ability_name = "salvage biomass"
 	keybind_signal = COMSIG_XENOABILITY_SALVAGE_PLASMA
+	cooldown_timer = DRONE_SALVAGE_COOLDOWN
+
+/datum/action/xeno_action/activable/salvage_biomass/on_cooldown_finish()
+	owner.playsound_local(owner, 'sound/voice/alien_drool1.ogg', 25, 0, 1)
+	to_chat(owner, "<span class='notice'>We are ready to salvage xeno biomass again.</span>")
```

Comment:
playsound(owner, sound)

Reply:
Better to make this local; I plan on refactoring all cooldown audio notifications to work like this in the near future.